$MirOS: ports/print/cups/patches/patch-scheduler_Makefile,v 1.2 2006/06/21 19:25:35 bsiegert Exp $
--- scheduler/Makefile.orig	Mon Sep 11 20:29:46 2006
+++ scheduler/Makefile	Wed Nov  8 18:15:12 2006
@@ -134,22 +134,6 @@ install:	all
 	echo Creating $(SERVERROOT)/ssl...
 	$(INSTALL_DIR) -m 700 $(SERVERROOT)/ssl
 	-chgrp $(CUPS_GROUP) $(SERVERROOT)/ssl
-	echo Creating $(STATEDIR)...
-	$(INSTALL_DIR) -m 755 $(STATEDIR)
-	echo Creating $(STATEDIR)/certs...
-	$(INSTALL_DIR) -m 511 $(STATEDIR)/certs
-	-chgrp $(CUPS_PRIMARY_SYSTEM_GROUP) $(STATEDIR)/certs
-	echo Creating $(LOGDIR)...
-	$(INSTALL_DIR) -m 755 $(LOGDIR)
-	echo Creating $(REQUESTS)...
-	$(INSTALL_DIR) -m 710 $(REQUESTS)
-	-chgrp $(CUPS_GROUP) $(REQUESTS)
-	echo Creating $(REQUESTS)/tmp...
-	$(INSTALL_DIR) -m 1770 $(REQUESTS)/tmp
-	-chgrp $(CUPS_GROUP) $(REQUESTS)/tmp
-	echo Creating $(CACHEDIR)...
-	$(INSTALL_DIR) -m 775 $(CACHEDIR)
-	-chgrp $(CUPS_GROUP) $(CACHEDIR)
 #	echo Creating $(CACHEDIR)/ppd...
 #	$(INSTALL_DIR) -m 755 $(CACHEDIR)/ppd
 #	-chgrp $(CUPS_GROUP) $(CACHEDIR)/ppd
@@ -186,13 +170,13 @@ uninstall:	all
 
 cupsd:	$(CUPSDOBJS) libmime.a ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o cupsd $(CUPSDOBJS) libmime.a \
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o cupsd $(CUPSDOBJS) libmime.a \
 		$(LIBZ) $(SSLLIBS) $(LIBSLP) $(LIBLDAP) $(PAMLIBS) \
 		$(LIBPAPER) $(LIBMALLOC) $(CUPSDLIBS) $(LIBS)
 
 cupsd-static:	$(CUPSDOBJS) libmime.a ../cups/libcups.a
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o cupsd-static $(CUPSDOBJS) libmime.a \
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o cupsd-static $(CUPSDOBJS) libmime.a \
 		$(LIBZ) $(SSLLIBS) $(LIBSLP) $(LIBLDAP) $(PAMLIBS) \
 		../cups/libcups.a $(COMMONLIBS) $(LIBZ) $(LIBPAPER) \
 		$(LIBMALLOC) $(CUPSDLIBS)
@@ -204,7 +188,7 @@ cupsd-static:	$(CUPSDOBJS) libmime.a ../
 
 cups-deviced:	cups-deviced.o util.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o cups-deviced cups-deviced.o util.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o cups-deviced cups-deviced.o util.o $(LIBS)
 
 
 #
@@ -213,7 +197,7 @@ cups-deviced:	cups-deviced.o util.o ../c
 
 cups-driverd:	cups-driverd.o util.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o cups-driverd cups-driverd.o util.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o cups-driverd cups-driverd.o util.o $(LIBS)
 
 
 #
@@ -222,7 +206,7 @@ cups-driverd:	cups-driverd.o util.o ../c
 
 cups-lpd:	cups-lpd.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o cups-lpd cups-lpd.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o cups-lpd cups-lpd.o $(LIBS)
 
 
 #
@@ -231,7 +215,7 @@ cups-lpd:	cups-lpd.o ../cups/$(LIBCUPS)
 
 cups-polld:	cups-polld.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o cups-polld cups-polld.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o cups-polld cups-polld.o $(LIBS)
 
 
 #
@@ -251,7 +235,7 @@ libmime.a:	$(MIMEOBJS)
 
 testdirsvc:	testdirsvc.o
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o testdirsvc testdirsvc.o $(COMMONLIBS) $(NETLIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o testdirsvc testdirsvc.o $(COMMONLIBS) $(NETLIBS)
 
 
 #
@@ -260,7 +244,7 @@ testdirsvc:	testdirsvc.o
 
 testlpd:	testlpd.o ../cups/libcups.a cups-lpd
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o testlpd testlpd.o ../cups/libcups.a \
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o testlpd testlpd.o ../cups/libcups.a \
 		$(COMMONLIBS) $(LIBZ) $(SSLLIBS)
 
 
@@ -270,7 +254,7 @@ testlpd:	testlpd.o ../cups/libcups.a cup
 
 testmime:	testmime.o libmime.a ../cups/libcups.a
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ testmime.o libmime.a ../cups/libcups.a \
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ testmime.o libmime.a ../cups/libcups.a \
 		$(COMMONLIBS) $(LIBZ) $(SSLLIBS)
 
 
@@ -280,7 +264,7 @@ testmime:	testmime.o libmime.a ../cups/l
 
 testspeed:	testspeed.o ../cups/libcups.a
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o testspeed testspeed.o ../cups/libcups.a \
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o testspeed testspeed.o ../cups/libcups.a \
 		$(COMMONLIBS) $(LIBZ) $(SSLLIBS)
 
 
@@ -290,7 +274,7 @@ testspeed:	testspeed.o ../cups/libcups.a
 
 testsub:	testsub.o ../cups/libcups.a
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o testsub testsub.o ../cups/libcups.a \
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o testsub testsub.o ../cups/libcups.a \
 		$(LIBGSSAPI) $(SSLLIBS) $(COMMONLIBS) $(LIBZ)
 
 
